src/content/app/genome-browser/components/drawer/drawer-views/variant-group-legend/VariantGroupLegend.scss

```diff
@@ -0,0 +1,47 @@
+@import 'src/styles/common';
+
+.container {
+  margin: 10px 0 0 80px;
+  width: 950px;
+}
+
+.label {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  margin-right: 10px;
+}
```

Comment:
What's the purpose of these three CSS rules?

Reply:
Not needed. removed